apps/web/components/booking/BookingListItem.tsx

```diff
@@ -430,6 +442,15 @@ function BookingListItem(booking: BookingItemProps) {
         setIsOpenDialog={setIsOpenAddGuestsDialog}
         bookingId={booking.id}
       />
+      <ReportBookingDialog
+        isOpenDialog={isOpenReportDialog}
+        setIsOpenDialog={setIsOpenReportDialog}
+        bookingId={booking.id}
+        isRecurring={isRecurring}
+        isUpcoming={isUpcoming}
+        isCancelled={isCancelled}
+        isRejected={isRejected}
```

Comment:
I wonder if we could fuse this into a single status prop instead of separate ones, especially because only one of them is true at a time 🤔